php/test/test_async.php

```diff
@@ -292,7 +289,7 @@ function test_symbol($exchange, $symbol, $code) {
             test_open_orders($exchange, $symbol);
             test_transactions($exchange, $code);
             $balance = yield $exchange->fetch_balance();
-            var_dump($balance);
+            var_dump("fetched balance entries: " . count(array_keys($balance)));
```

Comment:
no need to output very long response from balance test. we shouldnt do that in either JS/py

Reply:
The replacement is not entirely correct here, since the balance structure has at least a few extra fields apart from the currencies themselves. I'm not sure if printing 4 entries on an empty balance makes much sense.

```suggestion
```

Reply:
You are absolutely right, I've been thinking about it ofc, however, for quick resolution, I've made that, but actually I could ` - - 4` too. I've also thought already about what you suggest, but I still think that it's not needed to print-out all those token keys imho, and for test-scope, even the "successfully passed" message is enough, but for more clarity, we can add "amount" too, without taking screen much by those all keys (bcz some exchanges do have 500 balance items or etc, so we'd better to just print "fetched 500 balance items" imo).